src/custom-element.js

```diff
@@ -849,7 +849,8 @@ function createBaseCustomElementClass(win) {
           this.layout_ = applyLayout_(this);
           if (this.layout_ != Layout.NODISPLAY &&
```

Comment:
Let's have a different PR/Issue if we need to clean up `connectedCallback`. I am not exactly sure what needs go away here.